config/local_conda_mapping.yml.sample

```diff
@@ -1,2 +1,50 @@
 # See $GALAXY_ROOT/lib/galaxy/tools/deps/default_conda_mapping.yml for example mapping - 
 # additional site-specific mappings can be added to config/conda_mapping.yml.
+- from:
+    name: samtools
+    version: Null
+  to:
+    name: samtools
+    version: 1.3.1
+- from:
+    name: ucsc_tools
+    version: Null
+  to:
+    name: ucsc_tools
+    version: 332
+- from:
+    name: bedtools
+    version: Null
+  to:
+    name: bedtools
+    version: 2.26.0gx
+- from:
+    name: gnuplot
+    version: Null
+  to:
+    name: gnuplot
+    version: 4.6
+- from:
+    name: openbabel
+    version: 2.3.2
```

Comment:
There should be a 2.3.2 version: https://anaconda.org/bioconda/openbabel/files
- but only for Linux. In theory the 2.4 should work, but I think I need to test this before I feel really good about it.